protokube/pkg/protokube/tainter.go

```diff
@@ -68,7 +68,7 @@ func ApplyMasterTaints(kubeContext *KubernetesContext) error {
 		nodeTaintJSON := node.Annotations[v1.TaintsAnnotationKey]
 		if nodeTaintJSON != "" {
 			if nodeTaintJSON != string(taintJSON) {
-				glog.Infof("Node %q had unexpected taint: %v", node.Name, nodeTaintJSON)
+				glog.Infof("Node %q is registered with taint: %v", node.Name, nodeTaintJSON)
```

Comment:
Sure, so add something like --register-taints to ProtokubeFlags and toggle it in the template function?